test/test_crypto.py

```diff
@@ -211,6 +255,24 @@ def test_rsa_signing_roundtrip(self):
                 rsa_pub = RSA.new_public_key_from_pem_data(RSA_PUBLIC_KEY_PEM)
                 self.assertTrue(rsa_pub.verify(p, digest, signature))
 
+    def test_rsa_signing_roundtrip_pkcs8(self):
```

Comment:
trivial: test error-handling path by adding "negative" test that fails to find any private key